pyproject.toml

```diff
@@ -125,6 +127,8 @@ module = [
     "tlv8.*",
     "pytest.*",
     "click_aliases.*",
+    "smartcard.*",
```

Comment:
We already have typing stubs for `smartcard` and removing this override does not cause any errors for me locally, so why do we need this?

Reply:
Good catch. I'm removing this.